index.html

```diff
@@ -0,0 +1,88 @@
+                <nav class="nav-bar">
+                    <div class="nav-bar-item">
+                        <a href="#" title="logo">Welcome</a>
+                    </div>
+                    <div class="nav-bar-item">
+                        <img src="./static/images/hamburger-menu-icon.png" title="Menu Options" alt="Menu" width="30" height="30">
+                    </div>
+                </nav>
+            </header>
+
+            <div class="cont-preview flex-item">
+                <div>
+                    <header class="cont-header">
+                        <span>Hello there! I'm Steven.</span>
+                        <span>Software Developer</span>
+                    </header>
+                    <div class="cont-desc">
+                        I have been a Software Developer for 5 years now, working with leading tech companies in developing
+                        cutting-edge technologies. I'm great at customizing websites as well, creating suitable APIs that swiften
+                        and render testing and debugging easy.
+                        If you're looking to hire a software developer, well, I'm your guy.
+                    </div>
+                    <div class="lets-connect">
+                        <div class="socio-hub">
+                            <img src="./static/images/github-icon.png" title="GitHub" alt="GitHub logo">
+                        </div>
+                        <div class="socio-hub">
+                            <img src="./static/images/linkedin-icon.png" title="Linkedin" alt="Linkedin logo">
+                        </div>
+                        <div class="socio-hub">
+                            <img src="./static/images/2-fingers-icon.png" title="2fingers" alt="2fingers logo">
+                        </div>
+                        <div class="socio-hub">
+                            <img src="./static/images/twitter-icon.png" title="Twitter" alt="Twitter logo">
+                        </div>
+                        <div class="socio-hub">
+                            <img src="./static/images/m-icon.png" title="M" alt="M logo">
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="flex-item">
+                <div class="skill-set">
+                    <div>
+                        <label>Languages</label>
+                        <ul>
+                            <li>English <span class="skill-level">Advanced</span></li>
+                            <li>French <span class="skill-level">Intermediate</span></li>
+                            <li>Kinyarwanda <span class="skill-level">Native</span></li>
+                        </ul>
+                    </div>
+                    <div>
+                        <label>Frameworks</label>
+                        <ul>
+                            <li>Node.JS</li>
+                            <li>Angular</li>
```

Comment:
- [ ] As requested for this project, you should fit exactly the original design from [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988)
But, in your design, I can see that you add some features about yourself. 
![image](https://user-images.githubusercontent.com/71874425/164570994-ac9705ff-873f-4a94-8f0a-5c5c3dfdff64.png)

Kindly, remove this part from your project on this level. you'll add it when you'll reach the next project(where you'll add the `About` and `Works` sections). Once you implement your website, make sure that you follow exactly the figma design.

Reply:
Removed